src/test/StrategyVault.t.sol

```diff
@@ -381,4 +384,219 @@ contract SVaultTest is TestPlus {
+        // Initialize proxy with correct data
+        bytes memory initData = abi.encodeCall(
+            StrategyVault.initialize,
+            (governance, address(asset), "Affine High Yield LP - USDC-wETH", "affineSushiUsdcWeth")
+        );
+        ERC1967Proxy proxy = new ERC1967Proxy(address(impl), initData);
+        StrategyVault sVault = StrategyVault(address(proxy));
+
+        //add a dummy strategy
+        address[] memory strategists = new address[](1);
+        strategists[0] = address(this);
+        MockEpochStrategy strategy1 = new MockEpochStrategy(sVault, strategists);
+
+        vm.startPrank(governance);
+        sVault.setStrategy(strategy1);
+
+        uint256 initialAssets = 100_000 * (10 ** asset.decimals());
+        sVault.setTvlCap(10 * initialAssets);
+
+        console.log("initial assets %s deci %s", initialAssets, asset.decimals());
+        console.log("Debt escrow %s", address(sVault.debtEscrow()));
+
+        deal(address(asset), alice, initialAssets);
+
+        changePrank(alice);
+
+        MockERC20(sVault.asset()).approve(address(sVault), initialAssets);
+
+        sVault.deposit(initialAssets, alice);
+
+        sVault.transfer(address(sVault.debtEscrow()), 10 ** (sVault.decimals()));
+
+        changePrank(strategists[0]);
+
+        strategy1.endEpoch();
+    }
+
+    function testFailWithdrawWithNullEscrow() public {
+        // Deploy vault
+        StrategyVault impl = new StrategyVault();
+        // Initialize proxy with correct data
+        bytes memory initData = abi.encodeCall(
+            StrategyVault.initialize,
+            (governance, address(asset), "Affine High Yield LP - USDC-wETH", "affineSushiUsdcWeth")
+        );
+        ERC1967Proxy proxy = new ERC1967Proxy(address(impl), initData);
+        StrategyVault sVault = StrategyVault(address(proxy));
+
+        //add a dummy strategy
+        address[] memory strategists = new address[](1);
+        strategists[0] = address(this);
+        MockEpochStrategy strategy1 = new MockEpochStrategy(sVault, strategists);
+
+        vm.startPrank(governance);
+        sVault.setStrategy(strategy1);
+
+        uint256 initialAssets = 100_000 * (10 ** asset.decimals());
+        sVault.setTvlCap(10 * initialAssets);
+
+        console.log("initial assets %s deci %s", initialAssets, asset.decimals());
+        console.log("Debt escrow %s", address(sVault.debtEscrow()));
+
+        deal(address(asset), alice, initialAssets);
+
+        changePrank(alice);
+
+        MockERC20(sVault.asset()).approve(address(sVault), initialAssets);
+
+        sVault.deposit(initialAssets, alice);
+
+        changePrank(strategists[0]);
+
+        strategy1.beginEpoch();
+
+        changePrank(alice);
+
+        sVault.withdraw(initialAssets / 10, alice, alice);
+
+        changePrank(strategists[0]);
+
+        strategy1.endEpoch();
+    }
+}
+
+contract SVaultUpgradeLiveTest is SVaultTest {
+    function forkNet() public override {
```

Comment:
I don't think this inheritance is needed. We just want to run the one integration test below. We don't need to rerun all the tests from above.